hack/quick-install.sh

```diff
@@ -15,28 +18,4 @@ kubectl apply -f https://github.com/knative/eventing/releases/download/knative-$
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
-bash cmd/operator/generate.sh
-helm template kubearchive charts/kubearchive -n kubearchive \
-    --include-crds \
-    --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
-ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > /tmp/kubearchive.yaml
-kubectl apply -n kubearchive -f /tmp/kubearchive.yaml
-
-kubectl -n kubearchive rollout status deployment --timeout=90s
-kubectl -n kubearchive wait pod --all --for=condition=ready --timeout=90s
-
-# Make sure webhooks are up and running.
-LOCAL_PORT=8443
-kubectl -n kubearchive port-forward service/kubearchive-operator-webhooks ${LOCAL_PORT}:443 >& /dev/null &
-
-# Wait for $LOCAL_PORT to become available.
-while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
-    echo -n .
-    sleep 0.5
-done
-echo .
-
-kubectl get -n kubearchive deployments
-
-# Kill all background jobs, including the port-forward started earlier.
-trap 'kill $(jobs -p)' EXIT
+hack/kubearchive-install.sh
```

Comment:
The other call to a script in this code uses the syntax `bash <dir>/install.sh` (line 11), can you do the same here to make them consistent?

```suggestion
bash hack/kubearchive-install.sh
```

Reply:
Fixed.